src/turbo_flask/turbo.py

```diff
@@ -71,7 +71,11 @@ def turbo(self, version=_VER, url=None):
         if ws_route:
             return Markup(f'''<script type="module">
 import * as Turbo from "{url}";
-Turbo.connectStreamSource(new WebSocket(`ws://${{location.host}}{ws_route}`));
+try {
+    Turbo.connectStreamSource(new WebSocket(`ws://${location.host}/turbo-stream`));
+} catch(error) {
+    Turbo.connectStreamSource(new WebSocket(`wss://${location.host}/turbo-stream`));
+}
```

Comment:
It would be better to get the correct scheme from the page, I should have done that and forgot.

Reply:
Do you mean the request.scheme variable in Flask? The problem with that is that the client may be connecting with HTTPS to an Nginx proxy frontend and Nginx then speaks HTTP to the Flask backend, giving incorrect info about what scheme the browser is actually using.

Reply:
That problem has a solution, the `ProxyFix` middleware addresses that. But if you prefer, you can look at the scheme in the browser side.

Your solution is not good because it prefers ws:// over wss://.